Commit current state of exp example apps EXP-3667

brianfeister opened the pull request.

Commit:
Commit current state of exp example apps EXP-3667

adamgallowayscala commented:
@brianfeister socialmedia-app needs to be removed and the bower_components of weather-app needs to be removed

adamgallowayscala commented on README.md:
```diff
@@ -1,13 +1,57 @@
 | App Template | UUID |
 |--------------|------|
 |audio|3415bd0b-f567-4f53-896a-2344b24645be|
-|dayplan|4d0d80e8-fdba-46a2-8a54-90fdab96390d
```
why remove this and schedule?

brianfeister replied:
To be clear, I copy-pasted the README.md, and this was the state of current develop

adamgallowayscala replied:
ok, this should be merged from production

adamgallowayscala replied:
still probably broken there but we don't want app code from develop

brianfeister replied:
gotcha, doing that now

adamgallowayscala reviewed (changes requested):
also needs slideshow changed to playlist in the readme table of uuids

Commit:
Remove some weather-app bower_components and socialmedia-app EXP-3667

Commit:
update slideshow => playlist in README

brianfeister commented:
@adamgallowayscala Changes I made:

1. Copy-pasted the state of production (this had no impact, they are in sync)
1. Removed `bower_components` from weather app
1. Removed `socialmedia-app`
1. Updated README `slideshow` => `playlist`

adamgallowayscala committed:
Update README.md

adamgallowayscala commented:
👍
